ui/webui/src/components/HeaderKebab.jsx

```diff
@@ -152,6 +162,11 @@ export const HeaderKebab = () => {
               isModalOpen={isAboutModalOpen}
               setIsAboutModalOpen={setIsAboutModalOpen}
             />}
+            {isReportIssueOpen &&
+            <UserIssue
+              reportLinkURL={reportLinkURL}
+              setIsReportIssueOpen={setIsReportIssueOpen}
+            />}
```

Comment:
Probably fine for now, but I wonder if these `isSomethingOpen` variables are usable at scale? We'll eventually have many of these modals openable from the menu, right? Or am I reading this wrong and it's certain there won't be code doing `if (isAaOpen || isBbOpen || isCcOpen) ...`?

Reply:
Yes, I agree it's fine now, especially as I didn't find the spaghetti condition anywhere. That's also why I asked - I thought perhaps I'm missing something and the spaghetti can't happen, or is resolved automagically by the framework instead. (Which would be fine too!)